index.html

```diff
@@ -1,15 +1,62 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <title>Tamagotchi</title>
-    <link href="./styles.css" rel="stylesheet" />
-  </head>
-  <body>
-    <div class="gameBoy">
-      <p class="bar">Energy <span class="health" /></p>
-    </div>
-
-    <script src="./app.js" type="module"></script>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <title>Tamagotchi</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=VT323&display=swap" rel="stylesheet">
+  <link href="./styles.css" rel="stylesheet" />
+</head>
+
+<body>
+  <main class="main">
+    <section class="section">
+      <div class="gameBoy">
+        <div class="gameDisplay">
+          <div class="gameStats">
+            <ul class="gameParameters-list">
+              <li class="gameParameters-item">
+                <p class="bar"><img src="./images/heart.png" alt="heart-icon"> <span class="health"></span></p>
+              </li>
+              <li class="gameParameters-item">
+                <p class="bar"><img src="./images/apple.png" alt="apple-icon"> <span class="hunger"></span></p>
+              </li>
+              <li class="gameParameters-item">
+                <p class="bar"><img src="./images/sleeping.png" alt="sleeping-icon"> <span class="energy"></span></p>
+              </li>
+              <li class="gameParameters-item">
+                <p class="bar"><img src="./images/playing.png" alt="playing-icon"> <span class="fun"></span></p>
+              </li>
+            </ul>
+            <div class="tamagotchiFrame">
+              <img class="tamagotchiImage" src="./images/happyTamatgochi.png" alt="Tamagotchi whne it is happy">
+              <div class="tamagotchiLine"></div>
+            </div>
+            <span class="tamagotchiStates">Happy</span>
+          </div>
+        </div>
+        <div class="gameButtons">
+          <ul class="gameButtons-list">
+            <li class="gameButtonItem"><button class="gameButton"><img src="/images/apple.png"
```

Comment:
**issue:** These images srcs are not working correctly. It is always safer to make them relative: `src="./images/apple.png"`. Just as you did above in parameters icons:)